docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -18,139 +18,133 @@ This article covers how you can get started with using F# on .NET Core 1.0 with
-3. Change `Program.fs` to:
-
-    ```fs
-    open System
-    open Library
-
-    [<EntryPoint>]
-    let main argv = 
-        printfn "Nice command line arguments!  Here's what JSON.NET has to say about them:"
-
-        argv
-        |> Array.map getJsonNetJson
-        |> Array.iter (printfn "%s")
-
-        0 // return an integer exit code
-    ```
-
-4. Add a reference to the `Library` project you just created in the `project.json` file.  It should look like this:
-
-    ```json
-    {
-      "version": "1.0.0-*",
-      "buildOptions": {
-        "debugType":"portable",
-        "emitEntryPoint": true,
-        "compilerName": "fsc",
-        "compile": {
-          "includeFiles": [
-            "Program.fs"
-          ]
-        }
-      },
-      "tools": {
-        "dotnet-compile-fsc":"1.0.0-preview2-*"
-      },
-      "frameworks": {
-        "netcoreapp1.0": {
-          "dependencies": {
-            "Microsoft.NETCore.App": {
-              "type": "platform",
-              "version": "1.0.0"
-            },
-            "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629",
-            "Library":{
-              "target": "project"
-            }
-          }
-        }
-      }
-    }
-    ```
-
-10. Enter `dotnet restore` and `dotnet build` into the command line.  These should succeed.
-11. Enter `dotnet run Hello World` into the command line.  You should see results like this:
-
-```
-Nice command line arguments!  Here's what JSON.NET has to say about them:
+Use the `dotnet new` command, create a Console app in the **src** folder named App. 
+
+```bash
+dotnet new console -lang F# -o src/App 
+```
+
+The folowing directory structure is produced as a result of the command completing:
+
+```
+└── FSNEtCore
+    ├── FSNEtCore.sln
+    └── src
+        ├── App
+        │   ├── App.fsproj
+        │   ├── Program.fs
+        └── Library
+            ├── Library.fs
+            └── Library.fsproj
+```
+
+Change `Program.fs` to:
+
+```fsharp
+open System
+open Library
+
+[<EntryPoint>]
+let main argv = 
+    printfn "Nice command-line arguments!  Here's what JSON.NET has to say about them:"
+
+    argv
+    |> Array.map getJsonNetJson
+    |> Array.iter (printfn "%s")
+
+    0 // return an integer exit code
+```
+
+Change directories to the `App` console project and add a reference to the `Library` project using `dotnet add reference`.
+
+```bash
+dotnet add reference ../Library/Library.fsproj
+```
+Add the `Library` project to the `FSNetCore` solution using the `dotnet sln add` command:
+
+```bash
+dotnet sln add src/App/App.fsproj
+```
+
+Restore the NuGet dependencies, `dotnet restore` and run `dotnet build` to Build the project.
+
+Run the project passing `Hello World` as arguments.
+
+```bash
+dotnet run Hello World
+``` 
+
+You should see the following results:
+
+```
+Nice command-line arguments!  Here's what JSON.NET has to say about them:
 
 I used to be Hello but now I'm ""Hello""!
```

Comment:
is this the right output? the function above seems to have something like "thanks to JSON.NET" in the end.